haml_test.html.haml

```diff
@@ -0,0 +1,9 @@
+.main
+  .content
+    %div.hello
+      %p{ class: "testing"}
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should end with one space before the closing brace